src/components/KCard/KCard.vue

```diff
@@ -196,8 +196,11 @@ const showCardTitleWithStatus = computed((): boolean => useStatusHatLayout.value
 
   .k-card-title {
     margin-bottom: var(--kui-space-50, $kui-space-50) !important;
+    max-width: 80%;
```

Comment:
@portikM I'm not sure why this fix is needed now since we haven't touched KCard yet; is this broken on `main` as well?

Reply:
It is
![Screen Shot 2023-11-21 at 11 02 36 AM](https://github.com/Kong/kongponents/assets/36751160/0df3cfee-7d10-4e45-9928-ba8b1949f592)